src/main/java/io/output/Output.java

```diff
@@ -0,0 +1,115 @@
+package io.output;
+
+import coupon.Coupon;
+import cafe.Order;
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import menu.MenuCategory;
+import menu.MenuItem;
+
+public class Output {
+    private static final String HYPHEN = "-";
+
+    public static void printMenuItems(Map<MenuCategory, List<MenuItem>> menuItems) {
+        List<MenuCategory> menuCategories;
+        menuCategories = menuItems.keySet().stream().collect(Collectors.toList());
```

Comment:
```suggestion
        List<MenuCategory> menuCategories = new ArrayList<>(menuItems.keySet());
```

Set 타입을 stream api를 통해 List로 변환해서 초기화 하는 것 대신 Collection 끼리는 생성자 주입을 통해 타입을 변환할 수 있도록 내부적으로 구현되어 있습니다.
이를 Prototype Pattern이라고 하는데 여유되시면 공부해보시면 좋겠네요.!

Reply:
Prototype Pattern 공부해 보겠습니다!
감사합니다.